frontend/src/components/main/FutureProjectionForm.tsx

```diff
@@ -3,24 +3,35 @@ import { TextField, Button, Box } from '@mui/material';
 import { FutureProjection } from '@/types';
 
 const FutureProjectionForm: React.FC = () => {
-  const [futureProjection, setFutureProjection] = useState<FutureProjection>({
+  const [mainProjection, setMainProjection] = useState<FutureProjection>({
     id: 0,
     name: '',
-    mainProjection: '',
-    mainProjectionDescription: '',
-    alternativeProjection: '',
-    alternativeProjectionDescription: '',
+    projection: '',
```

Comment:
Was ist der Unterschied zwischen 'name' und 'projection'?

Reply:
ist in dem neuen Commit gefixt